README.md

```diff
@@ -1,31 +1,72 @@
 # 1Password Connect Go SDK
 
-The 1Password Connect Go SDK provides access to the 1Password Connect API hosted on your infrastructure. The library is intended to be used by your applications, pipelines, and other automations to simplify accessing items stored in your 1Password vaults.
+[![Go Reference](https://pkg.go.dev/badge/github.com/1Password/connect-sdk-go.svg)](https://pkg.go.dev/github.com/1Password/connect-sdk-go)
+[![Go Report Card](https://goreportcard.com/badge/github.com/1Password/connect-sdk-go)](https://goreportcard.com/report/github.com/1Password/connect-sdk-go)
+[![Version](https://img.shields.io/github/release/1Password/connect-sdk-go.svg)](https://github.com/1Password/connect-sdk-go/releases/)
 
-## Installation
+The 1Password Connect Go SDK provides access to the 1Password Connect API, to facilitate communication with the Connect server hosted on your infrastructure. The library is intended to be used by your applications, pipelines, and other automations to simplify accessing items stored in your 1Password vaults.
 
+## Installation
+To download and install the 1Password Connect Go SDK, as well as its dependencies:
 ```sh
 go get github.com/1Password/connect-sdk-go
 ```
 
 ## Usage
 
+Below, you can find a selection of the most used functionality of the Connect Go SDK. For more detailed information about the content of the SDK, please refer to the [GoDocs](https://pkg.go.dev/github.com/1Password/connect-sdk-go). 
```

Comment:
Are these go docs up to date?

Reply:
I'll address this before the end of the week. :D

Reply:
Go docs are updated with the latest version of Readme, as well as the code documentation.
At the moment of writing this comment, the documentation of the structs and functions is up to date (based on the latest release `v1.3.0`). 

The GoDocs readme will be updated when we make a new release with this version of README.